packages/visualizations/src/components/Table/Headers/SortButton.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+    import type { ColumnSortValues } from '../types';
+    import SortIcon from './SortIcon.svelte';
+
+    export let sorted: ColumnSortValues | undefined;
+</script>
+
+<button type="button" on:click>
+    <slot />
+    <SortIcon {sorted} />
+</button>
```

Comment:
I would also add an aria-label that can be customized with an optional `options` prop

Default values could be: Sort by ascending/descending order